cmd/syncthing/gui.go

```diff
@@ -1023,6 +1026,81 @@ func (s *apiService) getSystemLogTxt(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+type fileByte struct {
+	name string
+	buf  bytes.Buffer
+}
+
+func (s *apiService) getSupportBundle(w http.ResponseWriter, r *http.Request) {
+	var arrayFile []fileByte
+
+	jsonConfig, _ := json.MarshalIndent(getRedactedConfig(s), "", "  ")
+	arrayFile = addBufferFile(arrayFile, "config.json", jsonConfig)
+
+	jsonLog, _ := json.MarshalIndent(s.systemLog.Since(time.Time{}), "", "  ")
+	arrayFile = addBufferFile(arrayFile, "log.json", jsonLog)
+
+	jsonError, _ := json.MarshalIndent(s.guiErrors.Since(time.Time{}), "", "  ")
+	if string(jsonError) != "null" {
+		arrayFile = addBufferFile(arrayFile, "error.json", jsonError)
+	}
+
+	panicFiles, ok := checkPanicFile(baseDirs["config"])
+	if ok {
+		for _, f := range panicFiles {
+			panicF, _ := ioutil.ReadFile(f)
```

Comment:
Don't skip error handling here. If we get an error back the `panicF` will be null and bad things might happen. Just `if err != nil { continue }` would do.